objects/SetupChecker/script.lua

```diff
@@ -127,11 +127,11 @@ function onLoad(saved_data)
 
             -- queue up all dropdown changes as once
             Wait.frames(function()
-                local t = self.UI.getXmlTable()
-                t = updateAdversaryList(t)
-                t = updateScenarioList(t)
-                t = updateBoardLayouts(numPlayers, t)
-                self.UI.setXmlTable(t, {})
+                updateXml{
+                    updateAdversaryList(),
+                    updateBoardLayouts(numPlayers),
```

Comment:
is there any particular reason why you flipped scenario and board layout calls here?